### PR TITLE
Probs tests

### PR DESCRIPTION
**Pull request by antalszava**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 strawberryfields>=0.15
-pennylane>=0.19
+git+git://github.com/PennyLaneAI/pennylane
```
Comment (antalszava) [inline requirements.txt:2]: This change will be reverted right before merging. The idea was to check that tests pass with the latest version of PennyLane. Certain was were failing ([see here](https://github.com/PennyLaneAI/plugin-test-matrix/runs/4182515243?check_suite_focus=true#step:8:46)).
```suggestion
pennylane>=0.19
```

Comment (Jaybsoni) [reply requirements.txt:2]: Ahh makes sense

File: tests/test_tf.py [hunk 2]
```diff
@@ -513,7 +513,7 @@ def test_trace(self, tol):
         def circuit(x, y):
             qml.Squeezing(x, 0, wires=0)
             qml.Squeezing(y, 0, wires=1)
-            return qml.expval(qml.Identity(wires=[0, 1]))
+            return qml.expval(qml.Identity(wires=[0])), qml.expval(qml.Identity(wires=[1]))
```
Comment (Jaybsoni) [inline tests/test_tf.py:516]: Yup ! because it only takes a single wire now

**Review by Jaybsoni — APPROVED** [review]

**Comment by antalszava** [comment]
With #86 merged, closing this.

**Pull request closed without merging**